### PR TITLE
Use object url for image previews

### PR DESCRIPTION
**Pull request by keligijus**
Using FileReader for previews slows down the editor because it's using base64 encoded file. Object URL has better performance, the editor is no longer laggy or freezing while an image is being uploaded.

Please note, that I wasn't able to fix the broken test. If it's possible I would highly appreciate some guidance from the maintainers on fixing the test.

### CODE REVIEW

File: src/js/images.js [hunk 1]
```diff
@@ -285,13 +285,18 @@
             data.process().done(function () {
                 // If preview is set to true, let the showImage handle the upload start
                 if (that.options.preview) {
-                    reader = new FileReader();
-
-                    reader.onload = function (e) {
-                        $.proxy(that, 'showImage', e.target.result, data)();
+                    // using object URL instead of FileReader,
```
[the remark this answers is not in the document]
Comment (keligijus) [reply src/js/images.js]: Removed the comments.

File: .bowerrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+  {
```
[the remark this answers is not in the document]
Comment (keligijus) [reply .bowerrc]: Agreed. I added this because bower install would fail without, but it's irrelevant to this PR.

**Comment by keligijus** [comment]
Thanks for the test.

Changes are in the latest [commit](https://github.com/orthes/medium-editor-insert-plugin/pull/487/commits/9425d08bfb562dd7692cf3396ff8213cc56158fa). I hope it's not too messy with multiple commits.

**Comment by orthes** [comment]
thanks

**Comment by orthes** [comment]
@all-contributors please add @keligijus for code

**Pull request merged**